packages/wallet/src/redux/protocols/reducer-helpers.ts

```diff
@@ -174,6 +186,13 @@ export const channelIsClosed = (channelId: string, sharedData: SharedData): bool
   );
 };
 
+export const channelFundsAnotherChannel = (channelId: string, sharedData: SharedData): boolean => {
+  const latestCommitment = getLatestCommitment(channelId, sharedData);
+  return (
+    _.intersection(selectors.getChannelIds(sharedData), latestCommitment.destination).length > 0
```

Comment:
```suggestion
    !!_.intersection(selectors.getChannelIds(sharedData), latestCommitment.destination)
```

Reply:
Just a heads up that this doesn't work since `!![]` evaluates to `true`

Reply:
Right, that was a bit silly. !![].length does work, though I don't know if there's much benefit of doing that.